frontend/src/components/TrackingFieldsContent.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import {Descriptions} from "antd";
+import {withUser} from "../utils/withItem";
+
+export const TrackingFieldsContent = ({usersByID, entity}) => {
+    const displayUser = (user) => `${user.first_name} ${user.last_name} (${user.username})`
```

Comment:
Move this one outside of the component, otherwise you're creating a new function instance on every render.